doc/apimanager-reference.md

```diff
@@ -517,7 +517,7 @@ Note: Deploying databases internally with this section is meant for evaluation p
 
 | **Field** | **json/yaml field**| **Type** | **Required** | **Default value** | **Description** |
 | --- | --- | --- | --- | --- | --- |
-| Enabled | `enabled` | bool | No | `true` | Zync components are enabled by default. Set to false to disable all Zync components. If disabled, all other zync related settings will be ignored. |
+| Enabled | `enabled` | bool | No | `true` | Zync components are enabled by default. Set to false to disable all Zync components. If disabled, all other zync related settings will be ignored. When enabling from disabled, you must manually re-generate the routes if you wish to have the routes re-created |
```

Comment:
add link to knowledge base article https://access.redhat.com/solutions/6319751

Reply:
I was going to, but the article command doesn't work due to "deploymentConfigs" ...
Instead I've added the command with deployment to operator-user-guide. WDYT?